srv/adapter/ooba.ts

```diff
@@ -19,7 +19,7 @@ export const handleOoba: ModelAdapter = async function* (opts) {
 
   log.debug(`Prompt:\n${body.prompt}`)
 
-  const url = gen.thirdPartyUrl || user.koboldUrl
+  const url = user.oobaUrl
```

Comment:
This is a bug. This should definitely use the third party URL first if available.